docs/commands.rst

```diff
@@ -0,0 +1,213 @@
+Advanced: PyOTA Commands
+========================
+
+.. note::
+    **This page contains information about how PyOTA works under the hood.**
+
+    It is absolutely not necessary to be familiar with the content described
+    below if you just want to use the library.
+
+    However, if you are a curious mind or happen to do development on the
+    library, the following information might be useful.
+
```

Comment:
💭 Possibly useful to mention somewhere that PyOTA implements the [Command design pattern](https://en.wikipedia.org/wiki/Command_pattern).